editor/code_editor.cpp

```diff
@@ -222,6 +222,11 @@ void FindReplaceBar::_replace() {
 	} else {
 		text_editor->deselect(0);
 	}
+
+	// or we could also check replace button focus // if(replace->has_focus()){
```

Comment:
This kind of comment should go in the PR discussion, not in the code :)

Reply:
i remove comment now.